### PR TITLE
Run api and web in docker

### PR DESCRIPTION
**Pull request by verticalpalette**
This also includes quite a few updates to `vendor/`.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
+  api:
+    build:
+      context: .
+      dockerfile: ./ellies-pad/api/Dockerfile
```
Comment (annie1) [inline docker-compose.yml]: So this change gets everything working for Ellie's Pad. Do we want our new work to be under ./ellies-pad or something else? I don't really mind stomping on this, but figured you might want to keep it around.

File: ellies-pad/web/src/View.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import Relay from 'react-relay';
 import resetStyles from './resetStyles.js';
 import SavedSearch from './SavedSearch.js';
 
-export default class View extends React.Component {
+class View extends React.Component {
```
Comment (annie1) [inline ellies-pad/web/src/View.js]: Huh?

File: docker-compose.yml (hunk 1 above)
Comment (verticalpalette) [inline docker-compose.yml]: We'll keep working on Ellie's Pad.

File: ellies-pad/web/src/View.js (hunk 2 above)
Comment (verticalpalette) [inline ellies-pad/web/src/View.js]: Double export default. Weird that linter didn't catch it. Any way to make inline issues?

- [ ] Figure out why the linter didn't catch a duplicated default export.

Comment (verticalpalette) [inline ellies-pad/web/src/View.js]: - [] Why didn't the babel or the linter catch a duplicated default export?

**Pull request merged**